### ISSUE TITLE
Error: read ECONNRESET

### ISSUE THREAD

**Issue by twarped** [fent/node-ytdl-core#1226]
Hate to be that one guy... but I've been running into this problem starting today. It has to do with `ytdl.getInfo()`, and I've tried it with plenty of other videos, but it won't go away. Is it just a me thing, or is anyone else experiencing this?

**Comment by Nevah5** [comment]
I've also just got this issue. Don't have found anything to fix it yet. Will update you if I find something.

**Comment by twarped** [comment]
> I've also just got this issue. Don't have found anything to fix it yet. Will update you if I find something.

yeah, but i'm not exactly sure how to fix the problem, it will turn off after a couple of days tho. The error usually has to do with the server you're requesting, but it's not so much a wide spread issue yet... so just wait, maybe others will suffer too.

**Comment by twarped** [comment]
sweet, thanks.
